packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -763,47 +768,27 @@ export default {
     }
 }
 
-.c-nav-list-text {
```

Comment:
Now that this class has beeen deleted, you'll likely need to rename `c-nav-list-text-sub` as the naming won't make sense otherwise (as that indicates that `c-nav-list-text-sub` is a child of `c-nav-list-text`)